test/fixtures/app.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<app>
+<app width="667" height="1420">
```

Comment:
Is it good to reduce the image width/height in order to reduce the file size more?

Reply:
I wanted the image to be high enough resolution for testing image related functionality, it's about 900kb which i understand is a bit. what do you recommend?

Reply:
I just thought this image could be less size like https://github.com/appium/ruby_lib_core/blob/master/test/functional/data/test_has_blue.png is about 300kb (width 750 × height 1334)
(I usually put an image in 100kb on the web but this is kind of preference:))

Reply:
OK I reduced it to 250kb